social-meli/src/test/java/com/api/social_meli/unit/service/UserServiceTest.java

```diff
@@ -1,4 +1,102 @@
 package com.api.social_meli.unit.service;
 
+import com.api.social_meli.dto.GetFollowerCountDto;
+import com.api.social_meli.dto.MessageDto;
+import com.api.social_meli.exception.NotFoundException;
+import com.api.social_meli.model.User;
+import com.api.social_meli.repository.impl.UserRepositoryImpl;
+import com.api.social_meli.service.impl.UserServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith(MockitoExtension.class)
 public class UserServiceTest {
+
+    @Mock
+    private UserRepositoryImpl userRepository;
+
+    @InjectMocks
+    private UserServiceImpl userService;
+
+    //region T-0002 - US-0007 UNFOLLOW USER
+    @Test
+    void unfollowUser_success() {
```

Comment:
para mejora, hay que ver que nomenclatura vamos a usar para que los tests sean mas descriptivo. Por ejemplo si va a terminar siendo camelCase y con should, puede ser mas descriptivo y facil de entender que hace: shouldUnfollowUserSuccessfullyWhenBothUsersExist.